guides/getting-started.md

```diff
@@ -113,7 +113,7 @@ To install via npm, use the following command:
 npm install --save hammerjs
 ```
 
-After installing, import it on your app's root module.
+After installing, import it on your app's entry point (`src/main.ts`).
```

Comment:
I'd add an `e.g., ` before the filename to make it more clear that it's an example and that not everyone will necessarily have this file.